tests/Unit/Definition/LoaderTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace my127\Workspace\Tests\Unit\Definition;
+
+use PHPUnit\Framework\TestCase;
+
+class LoaderTest extends TestCase
+{
+    public function testNonExistantDefinition(): void
```

Comment:
```suggestion
    public function testNonExistentDefinition(): void
```

Reply:
Ah this was the original test case I was going to add :smile: I'll remove this from the PR